docs/conf.py

```diff
@@ -59,9 +59,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '3.2'
+version = '3.9'
 # The full version, including alpha/beta/rc tags.
-release = '3.2'
+release = '3.9.1'
```

Comment:
did you want this to be 4.0?

Reply:
Not until we close the release, so when we do a bump commit we'd just need to include this version here.